From 41f1f6e87bd9bed458e5391587f1e1b671d383bd Mon Sep 17 00:00:00 2001
From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Fri, 10 Jun 2022 12:31:10 +0900
Subject: [PATCH] test: drop redundant IMAGE_NAME=

If it is not specified, then "default" will be used.

(cherry picked from commit 3d52219390fa2d87938d99ec3bf71e10e101ded6)

Related: #2087652
---
 test/TEST-03-JOBS/test.sh         | 1 -
 test/TEST-17-UDEV/test.sh         | 1 -
 test/TEST-64-UDEV-STORAGE/test.sh | 1 -
 3 files changed, 3 deletions(-)

diff --git a/test/TEST-03-JOBS/test.sh b/test/TEST-03-JOBS/test.sh
index f827f90865..6a8d3a8a11 100755
--- a/test/TEST-03-JOBS/test.sh
+++ b/test/TEST-03-JOBS/test.sh
@@ -4,7 +4,6 @@ set -e
 
 TEST_DESCRIPTION="Job-related tests"
 TEST_NO_QEMU=1
-IMAGE_NAME="default"
 
 # shellcheck source=test/test-functions
 . "${TEST_BASE_DIR:?}/test-functions"
diff --git a/test/TEST-17-UDEV/test.sh b/test/TEST-17-UDEV/test.sh
index 079ecfd629..9d91eee204 100755
--- a/test/TEST-17-UDEV/test.sh
+++ b/test/TEST-17-UDEV/test.sh
@@ -3,7 +3,6 @@
 set -e
 
 TEST_DESCRIPTION="UDEV"
-IMAGE_NAME="default"
 TEST_NO_NSPAWN=1
 
 # shellcheck source=test/test-functions
diff --git a/test/TEST-64-UDEV-STORAGE/test.sh b/test/TEST-64-UDEV-STORAGE/test.sh
index c360c8b661..2a5b5b2dd3 100755
--- a/test/TEST-64-UDEV-STORAGE/test.sh
+++ b/test/TEST-64-UDEV-STORAGE/test.sh
@@ -9,7 +9,6 @@
 set -e
 
 TEST_DESCRIPTION="systemd-udev storage tests"
-IMAGE_NAME="default"
 TEST_NO_NSPAWN=1
 # Save only journals of failing test cases by default (to conserve space)
 TEST_SAVE_JOURNAL="${TEST_SAVE_JOURNAL:-fail}"
